tests/functional/test_upload_service.py

```diff
@@ -25,54 +25,142 @@ def __init__(self, *args, **kwargs):
         self.db_session_maker = DBSessionMaker()
 
     def setUp(self):
+        _start_time = time.time()
+        self.api_url = f"https://{os.environ['API_HOST']}/v1"
         self.upload_config = UploadConfig()
-        self.deployment_stage = os.environ['DEPLOYMENT_STAGE']
         self.auth_headers = {'Api-Key': self.upload_config.api_key}
-        self.api_url = f"https://{os.environ['API_HOST']}/v1"
+        self.deployment_stage = os.environ['DEPLOYMENT_STAGE']
+        self.upload_area_uuid = "deadbeef-dead-dead-dead-%012d" % random.randint(0, 999999999999)
         self.verbose = True
+        _end_time = time.time()
+        print(f"Total startup time: {_end_time - _start_time} seconds.")
 
-    def testIt(self):
+    def test_uploadSmallFile_successful(self):
+        # Test variables
+        _start_time = time.time()
+        _small_file = FixtureFile.factory('small_file')
+
+        # Run test
         print(f"\n\nUsing environment {self.deployment_stage} at URL {self.api_url}.\n")
+        self._execute_create_upload_area()
 
-        self.upload_area_uuid = "deadbeef-dead-dead-dead-%012d" % random.randint(0, 999999999999)
-        self._create_upload_area()
+        self._execute_upload_file_using_cli(_small_file.path)
+        self._verify_file_was_checksummed_inline(_small_file)
+
+        _validation_id = self._execute_validate_file(_small_file)
+        self._verify_file_validation_status(_validation_id)  # default parameters checks for success in validation
 
-        small_file = FixtureFile.factory('small_file')
+        self._execute_forget_upload_area()
+        self._execute_delete_upload_area()
 
-        self._upload_file_using_cli(small_file.path)
-        self._verify_file_was_checksummed_inline(small_file)
+        _end_time = time.time()
+        print(f"Total test_uploadSmallFile_successful time: {_end_time - _start_time} seconds.")
 
-        large_file = FixtureFile.factory('10241MB_file')
-        self._upload_file_using_cli(large_file.url)
-        self._verify_file_is_checksummed_via_batch(large_file)
+    def test_uploadLargeFile_successful(self):
+        # Test variables
+        _start_time = time.time()
+        _large_file = FixtureFile.factory('10241MB_file')
+
+        # Run test
+        print(f"\n\nUsing environment {self.deployment_stage} at URL {self.api_url}.\n")
+        self._execute_create_upload_area()
 
-        self._validate_file(small_file)
+        self._execute_upload_file_using_cli(_large_file.url)
+        self._verify_file_is_checksummed_via_batch(_large_file)
 
-        self._forget_upload_area()
-        self._delete_upload_area()
+        _validation_id = self._execute_validate_file(_large_file)
```

Comment:
I don't think we want to validate this large file in the functional test

Reply:
I don't think this was something I added? It was already getting validated...

Reply:
So looking at the function https://github.com/HumanCellAtlas/upload-service/blob/master/tests/functional/test_upload_service.py#L34, it looks like we only validate the small file. We do checksum both the small and large file

Reply:
Touché; updated.